stdlib/os/__init__.pyi

```diff
@@ -416,6 +417,16 @@ def getpid() -> int: ...
 def getppid() -> int: ...
 def strerror(__code: int) -> str: ...
 def umask(__mask: int) -> int: ...
+@final
+class uname_result(NamedTuple):
+    sysname: str
+    nodename: str
+    release: str
+    version: str
+    machine: str
+    n_fields: ClassVar[int]
+    n_sequence_fields: ClassVar[int]
+    n_unnamed_fields: ClassVar[int]
```

Comment:
These `n_foo` attributes are present on many classes, and currently handled a bit inconsistently:
- `pwd.pyi` has these as `ClassVar[int]`
- `time.pyi` defines these as a property, which would be on each instance of the class (this seems wrong)
- They are missing from several classes:

```
akuli@akuli-desktop:~/typeshed$ git grep n_sequence_fields tests | wc -l
22
```

It would be good to make them all consistent, but I would like to know whether other maintainers would prefer to have them or not have them. They don't seem very useful from a user's point of view.

Reply:
I agree they don't seem like particularly useful attributes, but I also don't see any _harm_ in having them. Personally, I think I'd prefer adding these attributes to 22 classes rather than having 22 wontfix entries in the allowlists? (But I'm obviously not a maintainer, so  it's obviously not my call :) )

Reply:
I've reverted the change, as I agree that we should either add these attributes to all 22 classes together, or not at all -- either way, it shouldn't be done in this PR :)